p4-16/psa/PSA.mdk

```diff
@@ -199,6 +202,41 @@ copy going to the control CPU to have a software-defined header
 (defined in the P4 program) that is different than any headers on the
 packet copies going to the Egress control block.
 
+A PSA implementation may choose to implement multiple class-of-service
+queues for each output port, but it may also choose to implement only
+1 queue for each output port.  The Ingress control block may choose to
+assign a value to the `ostd.class_of_service` field to change the
+packet's class of service to a value other than the default of 0.
+
+PSA only specifies how the Ingress control block can choose among
+multiple class-of-service queues for an output port, if more than one
```

Comment:
I would remove "for an output port" and replace "that might exist on the device".

Reply:
I have changed the text not to mention the queueing structure, except that a PSA implementation might implement different FIFO queues for different classes of service.